podcasts/Zendesk/MessageSupportViewModel.swift

```diff
@@ -121,18 +122,27 @@ class MessageSupportViewModel: ObservableObject {
 
     // MARK: Events
 
-    open func submitRequest() {
+    open func submitRequest(ignoreUnavailableWatchLogs: Bool = false) {
         isWorking.toggle()
+
         config.customFields(forDisplay: false, optOut: UserDefaults.standard.debugOptedOut)
             .flatMap { [unowned self] customFields -> AnyPublisher<String, Error> in
-                let requestObject = ZDSupportRequest(subject: self.config.subject,
-                                                     name: self.requesterName,
-                                                     email: self.requesterEmail,
-                                                     comment: self.comment,
-                                                     customFields: customFields,
-                                                     tags: self.config.tags)
-
-                return self.supportService.submitSupportRequest(requestObject)
+
+                // Check if the user mentioned watch on their issue description and if there
+                // are any Apple Watch logs available.
+                if self.comment.lowercased().contains("watch") && customFields.first(where: { $0.value.contains(FileLog.noWearableLogsAvailable) }) != nil && !ignoreUnavailableWatchLogs {
```

Comment:
Wdyt about also checking for the localized version of the word watch?

```suggestion
                let containsWatch = self.comment.localizedCaseInsensitiveContains(L10n.watch) || self.comment.lowercased().contains("watch")
                if containsWatch && customFields.first(where: { $0.value.contains(FileLog.noWearableLogsAvailable) }) != nil && !ignoreUnavailableWatchLogs {


```